python/pfs/drp/stella/fitBroadbandSED.py

```diff
@@ -61,11 +70,12 @@ class FitBroadbandSEDTask(Task):
     ConfigClass = FitBroadbandSEDConfig
     _DefaultName = "fitBroadbandSED"
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, **kwargs) -> None:
```

Comment:
The `-> None` is implied if it's not there, so you can leave it off.

Reply:
I don't think so. If no return type is specified, `-> Any` is implied. What is worse, if none of the parameter types nor the return type is specified just as this `__init__()` function, then `mypy` does not  do type checking on the function body in my experience.

Reply:
By running mypy on this program, you can see how mypy skips a function whose signature is not specified.
```
def f():
    return 1 + "2"
```
```
$ mypy a.py
Success: no issues found in 1 source file
```